### ISSUE TITLE
wrongly calculated t-SNE

### ISSUE THREAD

**Issue by oh-yu** [oh-yu/domain-invariant-learning#10]
```
✗
    target_feature_tsne = tsne.fit_transform(target_feature)
    source_feature_tsne = tsne.fit_transform(source_feature)
```

```
○
N_target = target_feature.shape[0]
feature_tsne = tsne.fit_transform(np.concatenate([target_feature, source_feature], axis=0))
target_feature_tsne, source_feature_tsne = feature_tsne[:N_target], feature_tsne[N_target:]
```

**Comment by oh-yu** [comment]
resolved by b28ab588604865102d679ca0dc686ed98ff3a58c

**Comment by oh-yu** [comment]
resolved by b28ab588604865102d679ca0dc686ed98ff3a58c
